src/archiefvernietigingscomponent/destruction/service.py

```diff
@@ -146,6 +146,19 @@ def get_resultaat(zaak_url: str) -> Optional[dict]:
     return resultaat
 
 
+def fetch_resultaat(resultaat_url: str) -> dict:
```

Comment:
this function and `get_resultaat` share a lot of code. Could you use this function inside `get_resultaat` to make it DRYer?